### PR TITLE
Adding build script curr_ubi_8.3.sh

### PR DESCRIPTION
**Pull request by asellappen**

### CODE REVIEW

**Comment by asellappen** [comment]
Test 1:

sh 
Cloning into 'curr'...
Already on 'main'
Your branch is up to date with 'origin/main'.
 found to checkout
=== RUN   TestLine
  Because Line() should provide current line
--- PASS: TestLine (0.00s)
=== RUN   TestFunc
  Because Func() should provide current function name
--- PASS: TestFunc (0.00s)
=== RUN   TestFile
  Because File() should provide current file name
--- PASS: TestFile (0.00s)
=== RUN   TestBasename
  Because Basename() should provide current file basename
--- PASS: TestBasename (0.00s)
=== RUN   TestDir
  Because Dir() should provide current directory
--- PASS: TestDir (0.00s)
PASS
coverage: 100.0% of statements
ok      github.com/otiai10/curr 0.003s  coverage: 100.0% of statements
 Build & Test Success..

**Comment by asellappen** [comment]
sh curr_ubi_8.3.sh v1.0.0

Test 2:

 sh curr_ubi_8.3.sh v1.0.0


  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 88e2f2c Merge pull request #3 from otiai10/develop
 found to checkout
/root/curr
=== RUN   TestLine
  Because Line() should provide current line
--- PASS: TestLine (0.00s)
=== RUN   TestFunc
  Because Func() should provide current function name
--- PASS: TestFunc (0.00s)
=== RUN   TestFile
  Because File() should provide current file name
--- PASS: TestFile (0.00s)
=== RUN   TestBasename
  Because Basename() should provide current file basename
--- PASS: TestBasename (0.00s)
=== RUN   TestDir
  Because Dir() should provide current directory
--- PASS: TestDir (0.00s)
PASS
coverage: 100.0% of statements
ok      github.com/otiai10/curr 0.004s  coverage: 100.0% of statements
 Build & Test Success..
[root@5e8cf58534a4 ~]#

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

File: c/curr/curr_ubi_8.3.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+if [ -d "curr" ] ; then
+  rm -rf curr
+fi
+# make sure to install make go under /root
+cd
+sudo dnf install -y git make wget tar gcc
+wget https://golang.org/dl/go1.16.7.linux-ppc64le.tar.gz
+tar -xvf go1.16.7.linux-ppc64le.tar.gz
+export PATH=/root/go/bin:$PATH
+export GOPATH=/root/go/bin
```
[the remark this answers is not in the document]
Comment (asellappen) [reply c/curr/curr_ubi_8.3.sh:37]: taken care

File: c/curr/curr_ubi_8.3.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+if [ -d "curr" ] ; then
+  rm -rf curr
+fi
+# make sure to install make go under /root
+cd
+sudo dnf install -y git make wget tar gcc
+sudo wget https://golang.org/dl/go1.16.7.linux-ppc64le.tar.gz
+sudo tar -xvf go1.16.7.linux-ppc64le.tar.gz
+export PATH=/root/go/bin:$PATH
+export GOPATH=/root/go/bin
```
Comment (gururajrkatti) [inline c/curr/curr_ubi_8.3.sh:37]: We should not be hardcoding the path like this ? Also, we should be using non-root user as per the header comment

File: c/curr/curr_ubi_8.3.sh [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=main
```
Comment (gururajrkatti) [inline c/curr/curr_ubi_8.3.sh:23]: Can we set the default to "1.0.0" which is the tested version for this script

**Comment by asellappen** [comment]
sh    curr_ubi_8.3.sh

Note: switching to 'v1.0.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 88e2f2c Merge pull request #3 from otiai10/develop
v1.0.0 found to checkout
=== RUN   TestLine
  Because Line() should provide current line
--- PASS: TestLine (0.00s)
=== RUN   TestFunc
  Because Func() should provide current function name
--- PASS: TestFunc (0.00s)
=== RUN   TestFile
  Because File() should provide current file name
--- PASS: TestFile (0.00s)
=== RUN   TestBasename
  Because Basename() should provide current file basename
--- PASS: TestBasename (0.00s)
=== RUN   TestDir
  Because Dir() should provide current directory
--- PASS: TestDir (0.00s)
PASS
coverage: 100.0% of statements
ok      github.com/otiai10/curr 0.004s  coverage: 100.0% of statements
 Build & Test Success..

File: c/curr/curr_ubi_8.3.sh (hunk 2 above)
Comment (asellappen) [reply c/curr/curr_ubi_8.3.sh]: correct,taken care

**Pull request merged**